docs/operator-guide/upgrade/upgrade-edp-3.10.md

```diff
@@ -0,0 +1,347 @@
+    ```
+
+2. (Optional) If you are using the **Gerrit Operator**, update the necessary CRDs for Gerrit. Run the following command to apply the necessary CRDs to the cluster:
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/epam/edp-gerrit-operator/v2.22.0/deploy-templates/crds/v2.edp.epam.com_gerritgroupmembers.yaml
+    kubectl apply -f https://raw.githubusercontent.com/epam/edp-gerrit-operator/v2.22.0/deploy-templates/crds/v2.edp.epam.com_gerritgroups.yaml
+    kubectl apply -f https://raw.githubusercontent.com/epam/edp-gerrit-operator/v2.22.0/deploy-templates/crds/v2.edp.epam.com_gerritmergerequests.yaml
+    kubectl apply -f https://raw.githubusercontent.com/epam/edp-gerrit-operator/v2.22.0/deploy-templates/crds/v2.edp.epam.com_gerritprojectaccesses.yaml
+    kubectl apply -f https://raw.githubusercontent.com/epam/edp-gerrit-operator/v2.22.0/deploy-templates/crds/v2.edp.epam.com_gerritprojects.yaml
+    kubectl apply -f https://raw.githubusercontent.com/epam/edp-gerrit-operator/v2.22.0/deploy-templates/crds/v2.edp.epam.com_gerritreplicationconfigs.yaml
+    kubectl apply -f https://raw.githubusercontent.com/epam/edp-gerrit-operator/v2.22.0/deploy-templates/crds/v2.edp.epam.com_gerrits.yaml
+    ```
+
+3. (Optional) Update Tekton Pipeline labels:
+
+    In version 3.10, Tekton pipelines require specific labels indicating their type: **build**, **review**, **clean**, or **deploy**. These labels enable **Portal UI** to correctly display pipeline data.
+
+    Below are examples of the required labels for each pipeline type:
+
+    - **Build Pipeline**
+        ```yaml
+        metadata:
+          name: build
+          labels:
+            app.edp.epam.com/pipelinetype: build
+        ```
+    - **Review Pipeline**
+        ```yaml
+        metadata:
+          name: review
+          labels:
+            app.edp.epam.com/pipelinetype: review
+        ```
+    - **Clean Pipeline**
+        ```yaml
+        metadata:
+          name: clean
+          labels:
+            app.edp.epam.com/pipelinetype: clean
+        ```
+    - **Deploy Pipeline**
+        ```yaml
+        metadata:
+          name: deploy
+          labels:
+            app.edp.epam.com/pipelinetype: deploy
+        ```
+
+    :::note
+    For any custom Tekton pipelines created in earlier versions, ensure that each pipeline has the appropriate label to maintain compatibility.
+    :::
+
+    To label an existing custom pipeline, you can use the following command as a reference:
+
+    ```bash
+    kubectl label pipeline <pipeline-name> app.edp.epam.com/pipelinetype=<build | review | deploy> -n <krci-namespace>
+    ```
+
+4. (Optional) If you are using the **External Secret Operator** to manage secrets, specify a new field in the `values.yaml` file: `manageGitProviderSecretsName`. This field should be defined as follows:
```

Comment:
please add more information. So previously we use only one parameter store object and now u need to migrate git providers u need to migrate this parameters to new approach...